docs/data/material/discover-more/related-projects/related-projects.md

```diff
@@ -28,8 +28,17 @@ Feel free to submit a pull request!
 - [mui-image](https://mui-image.surge.sh): The only Material UI image component to satisfy the Material Design guidelines for loading images.
 - [material-ui-image](https://mui.wertarbyte.com/#material-ui-image): Adds a "materializing" effect to images so they fade in like [Material's image loading pattern](https://material.io/archive/guidelines/patterns/loading-images.html#loading-images-usage) suggests.
 
+### Chips
+
+- [mui-chips-input](https://viclafouch.github.io/mui-chips-input): A chips input designed for the React library MUI.
+
```

Comment:
Why does the community need this chip component? I mean, it seems to duplicate with https://mui.com/material-ui/react-autocomplete/#multiple-values.

Reply:
Hey @oliviertassinari ! Thanks for your question, it's a little bit different because it's not an autocomplete. Here, it's just a simple input where you have less logic...